lib/widgets/recent_dm_conversations.dart

```diff
@@ -114,6 +117,7 @@ class RecentDmConversationsItem extends StatelessWidget {
           child: Center(
             // TODO(#95) need dark-theme color
             child: Icon(ZulipIcons.group_dm, color: Colors.black.withOpacity(0.5))));
+        isBot = false;
```

Comment:
Why is this always false? Isn't it possible to create DMs with multiple bots?

Reply:
> In `RecentDmConversationsPage`, I wasn't sure whether to add (and where to add) the bot icon to a group dm where there may be one or multiple bot recipients.

As I mentioned in the PR description above, I wasn't quite sure about this. There are two cases in group DMs with bot users: 

- A group DM with both real and bot users; in which I think it is not reasonable to add a bot icon at the end of the label as we do it in `1:1` DM with bot user. One approach we can take is to add a bot icon after the name of each bot user, something like this: **Welcome Bot 🤖, Mahmood, S Muhmood**. Or we can ignore the bot icon at all. I am not sure.
- A group DM with just bot users; in which I think we can add a bot icon at the end of the label, but again, it will be confused with the first case, is it that the group DM is with multiple real users and one last bot user or the group DM is with all bot users?! If we ignore the bot icon in the first case, then this cannot be confusing!

Let's also hear from @gnprice about this!

Reply:
This is a question about how the UI should behave for users, so it's a good one to get feedback on from @alya .

Reply:
The bot icon should be associated with a particular user, not with a conversation as a whole. So I would expect it to appear after each bot's name, regardless of how many bot and non-bot users there are in the conversation.

Reply:
By the way, https://github.com/zulip/zulip/issues/26831 is the corresponding web app issue.

Reply:
That makes a lot of sense. Thanks for the clarification @alya!